css/css-overscroll-behavior/META.yml

```diff
@@ -0,0 +1,6 @@
+spec: https://drafts.csswg.org/css-overscroll-behavior/
+suggested_reviewers:
+  - bgirard
```

Comment:
Not sure if @bgirard has signed up to do reviews. May want to confirm with them first.